<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>818</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Subestaciones</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table03">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>value</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table04">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>761</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>730</y>
      <width>701</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo_cesel.PNG</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>730</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Progreso:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Caracteristicas:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="button03">
       <property name="text">
        <string>boton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button01">
       <property name="text">
        <string>bay_up</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button05">
       <property name="text">
        <string>bay_down</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button06">
       <property name="text">
        <string>bay_type_3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button02">
       <property name="text">
        <string>show_data</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button04">
       <property name="text">
        <string>plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Funciones:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Posición:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prev">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icono_retroceder.png</normaloff>../icono_retroceder.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="button_next">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icono_avanzarr.jpg</normaloff>../icono_avanzarr.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="borrar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>2</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>borrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icono_borrar.jpg</normaloff>../icono_borrar.jpg</iconset>
    </property>
    <property name="Borrar" stdset="0">
     <iconset>
      <normaloff>../icono_borrar.jpg</normaloff>../icono_borrar.jpg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInicio">
    <property name="title">
     <string>Inicio</string>
    </property>
    <addaction name="actionParametros_generales"/>
    <addaction name="actionCondiciones_ambientales"/>
    <addaction name="actionInformaci_n_del_sistema"/>
    <addaction name="actionResponsables"/>
    <addaction name="actionDimensiones_de_SE"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="actionExportar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuC_lculos">
    <property name="title">
     <string>Generar</string>
    </property>
    <widget class="QMenu" name="menuHojas_de_c_lculo">
     <property name="title">
      <string>Hojas de cálculo</string>
     </property>
     <widget class="QMenu" name="menuCables_de_media_tensi_n">
      <property name="title">
       <string>Cables de media tensión</string>
      </property>
      <addaction name="actiondistribuci_n_de_temperatura"/>
      <addaction name="actionAmpacidad"/>
     </widget>
     <addaction name="actionDistancias_de_seguridad"/>
     <addaction name="actionApantallamiento"/>
     <addaction name="actionAislamiento"/>
     <addaction name="actionPararrayos"/>
     <addaction name="actionCables_de_baja_tension"/>
     <addaction name="menuCables_de_media_tensi_n"/>
     <addaction name="actionTableros_electricos"/>
    </widget>
    <widget class="QMenu" name="menuPlanos">
     <property name="title">
      <string>Planos</string>
     </property>
     <addaction name="actionDiagrama_Unifilar"/>
    </widget>
    <addaction name="menuHojas_de_c_lculo"/>
    <addaction name="menuPlanos"/>
   </widget>
   <addaction name="menuInicio"/>
   <addaction name="menuC_lculos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionParametros_generales">
   <property name="text">
    <string>Información del proyecto</string>
   </property>
  </action>
  <action name="actionCondiciones_ambientales">
   <property name="text">
    <string>Condiciones ambientales</string>
   </property>
  </action>
  <action name="actionInformaci_n_del_sistema">
   <property name="text">
    <string>Parámetros eléctricos</string>
   </property>
  </action>
  <action name="actionResponsables">
   <property name="text">
    <string>Responsables</string>
   </property>
  </action>
  <action name="actionDimensiones_de_SE">
   <property name="text">
    <string>Parámetros dimensionales SE</string>
   </property>
  </action>
  <action name="actionApantallamiento">
   <property name="text">
    <string>Apantallamiento</string>
   </property>
  </action>
  <action name="actionDistancias_de_seguridad">
   <property name="text">
    <string>Distancias de seguridad</string>
   </property>
  </action>
  <action name="actionAislamiento">
   <property name="text">
    <string>Aislamiento</string>
   </property>
  </action>
  <action name="actionPararrayos">
   <property name="text">
    <string>Pararrayos</string>
   </property>
  </action>
  <action name="actionCables_de_baja_tension">
   <property name="text">
    <string>Cables de baja tension</string>
   </property>
  </action>
  <action name="actionTableros_electricos">
   <property name="text">
    <string>Tableros electricos</string>
   </property>
  </action>
  <action name="actionAmpacidad">
   <property name="text">
    <string>Ampacidad</string>
   </property>
  </action>
  <action name="actiondistribuci_n_de_temperatura">
   <property name="text">
    <string>Distribución de temperatura</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="actionGenerar_todo">
   <property name="text">
    <string>Generar todo</string>
   </property>
  </action>
  <action name="actionDiagrama_Unifilar">
   <property name="text">
    <string>Diagrama Unifilar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
